docs/local-setup/create-account.md

```diff
@@ -21,9 +33,19 @@ Usernames can be 5-32 characters long and contain any of the following:
 
 From here, you can decide how you would like to manage your private keys. There are two options: If you'd like to be able to recover your private key through NEAR should you lose it, enter your phone number. Otherwise, backup your private key manually. 
 
+### Recover your Private Keys via Phone Number
+
+After you have provided us with your phone number and clicked the "PROTECT ACCOUNT" button, you will be sent a 6 digit security code to your number. Please provide the same code on the next screen:
+
+![Step 3: Security Code](assets/entercode.png)
+
+Once you click on the "PROTECT ACCOUNT" button, you will be forwarded to the dashboard.
+
+![Wallet](assets/finalscreen.png)
+
 ### Backup your Private Keys Manually
 
-To so click below the the phone number entry on "I choose to backup my account manually".
+Alternatively, you can choose to backup your private keys manually. To do so click below the the phone number entry on "I choose to backup my account manually".
```

Comment:
in this case we're not backing up "private keys" actually, we're guaranteeing that our account can be recovered using a different (ie. not phone SMS) method which is using a seed phrase

the user will not b